bulk_lookup/forms.py

```diff
@@ -0,0 +1,116 @@
+import logging
+
+from django import forms
+from django.conf import settings
+from django.utils.crypto import get_random_string
+
+import stripe
+from ukpostcodeutils.validation import is_valid_postcode
+
+from .models import OutputOption
+
+
+logger = logging.getLogger(__name__)
+
+
+class CSVForm(forms.Form):
+    original_file = forms.FileField()
+
+
+class PostcodeFieldForm(forms.Form):
+    # This is hidden by default and only shown if the CSV fails validation
+    skip_bad_rows = forms.BooleanField(
```

Comment:
I guess it's more just to confirm you know those rows won't be processed. This dates from when the charge was based upon how many rows would be processed, which we didn't go with in the end, so could be dropped, though might then lead to user support as to why they weren't processed. Will ticket to improve later